components/blitz/resources/omero/FS.ice

```diff
@@ -398,6 +399,7 @@ module omero {
                                 int blockSize,
                                 bool ignoreSysFiles,
                                 bool ignoreDirEvents,
+                                bool platformCheck,
```

Comment:
If you do need to change the API, perhaps mention on https://trello.com/c/YcdKrVc9/234-document-breaking-changes-for-5-1.